test/check-exports/test.mjs

```diff
@@ -0,0 +1,109 @@
+  'whatwg-url', 'fill-range', 'eslint-scope', 'tr46', 'micromatch',
+  'convert-source-map', 'define-property', 'agent-base', 'shebang-regex',
+  'shebang-command', 'mimic-fn', 'globby', 'npm-run-path', 'mime',
+  '@babel/code-frame', 'extend-shallow', 'to-regex-range', 'onetime',
+  'https-proxy-agent', 'y18n', 'buffer', 'strip-bom', 'is-glob',
+  'doctrine', 'picocolors', 'pkg-dir', '@babel/types',
+  'regenerator-runtime', 'human-signals', '@jridgewell/trace-mapping',
+  'ignore', 'jsesc', 'parse-json', 'jest-worker', 'graceful-fs',
+  'jest-util', 'jsonfile', 'normalize-path', 'strip-json-comments',
+  'cosmiconfig', 'minimist', 'path-type', '@babel/parser', 'balanced-match',
+  'picomatch', 'typescript', 'isexe', 'statuses', 'entities', 'bytes',
+  'node-fetch', 'http-errors', '@babel/highlight',
+  '@babel/helper-validator-identifier', 'function-bind',
+  'async', 'sprintf-js', '@babel/generator', 'is-extendable',
+  'get-intrinsic', 'lodash', 'mime-db', 'source-map-support', 'mime-types',
+  'is-arrayish', '@babel/core', 'once', 'anymatch', 'depd',
+  'hosted-git-info', 'path-to-regexp', 'axios', 'is-core-module',
+  '@babel/template', 'cookie', 'write-file-atomic', 'js-tokens',
+  '@typescript-eslint/typescript-estree', '@typescript-eslint/types',
+  'object-inspect', 'wrappy', 'is-extglob', 'chokidar',
+  '@typescript-eslint/visitor-keys', 'call-bind', 'loader-utils',
+  'browserslist', 'http-proxy-agent', 'fast-glob', 'concat-map',
+  'inflight', 'ajv-keywords', 'ansi-escapes', 'ci-info', 'fast-deep-equal',
+  'caniuse-lite', 'fs.realpath', '@jridgewell/gen-mapping', 'setprototypeof',
+  'strip-final-newline', 'optionator', 'path-is-absolute', '@babel/traverse',
+  'core-util-is', 'has-symbols', 'yocto-queue', 'p-try',
+  'electron-to-chromium', '@smithy/smithy-client', 'yaml', 'ini',
+  '@babel/helper-plugin-utils', 'jest-get-type', 'type-check', 'levn',
+  'is-descriptor', 'prelude-ls', 'slice-ansi',
+  '@typescript-eslint/scope-manager', 'isobject', 'esprima',
+  '@babel/helper-split-export-declaration', 'callsites', 'readdirp',
+  'escalade', 'import-fresh', 'get-caller-file', '@jridgewell/sourcemap-codec',
+  'acorn-walk', 'rxjs', 'ieee754', 'is-plain-obj',
+  'istanbul-lib-instrument', '@babel/helper-module-imports', 'side-channel',
+  'normalize-package-data', 'is-plain-object', 'util-deprecate',
+  '@jridgewell/resolve-uri', 'follow-redirects', 'array-union',
+  'json-parse-even-better-errors', 'path-parse', 'has-property-descriptors',
+  'uri-js', 'safer-buffer', '@babel/helpers', 'on-finished',
+  '@babel/helper-function-name', 'p-map', 'postcss-value-parser', 'indent-string',
+  '@babel/helper-module-transforms', 'object-assign', 'delayed-stream',
+  '@nodelib/fs.stat', 'require-directory', 'diff', 'parse5',
+  'asynckit', 'tmp', 'combined-stream']
+
+console.log(`📦  Installing ${modules.length} libraries...`)
+installLibs(modules)
+
+for (const mod of modules) {
+  testLib(mod)
```

Comment:
I'd rather if we returned `errored` from `testLib` and mutate it there, and then move the `let errored` declaration down here.

Makes the side effect easier to understand (and you could even do a `array.map().some()` call too).

Reply:
Yeah, can switch it around. The main thing I wanted was for the tests to continue on failures